source/migrating_from_legacy_products/index.html.md.erb

```diff
@@ -3,6 +3,121 @@ title: Migrate from our old frameworks
 weight: 210
 ---
 
-# Migrate from our old frameworks
+# Migrate to GOV.UK Frontend from our old frameworks
```

Comment:
Ah very true! But just to check - do you mean 'a prototype that uses Frontend'? Or 'an app that uses the Prototype Kit'?

Reply:
I'm actually thinking both.

Updating old and large versions of the kit can currently be pretty time consuming, we already know that and we want to make that process better. However meanwhile maybe we should encourage them to start a new prototype? Also not even sure if users would even consider using this guide to make that update but they might do...

If your app is a prototype that uses the legacy frameworks and you want to update it to test some new changes for the live website then I'm not sure what we should advise them to do. Perhaps this is not a likely scenario.

I realise that I'm offering more questions than answers here...

Reply:
No worries - on your first point, I've added a line in the next commit that warns Prototype Kit users not to migrate.

On your second point - I think that, as you say, this may not be a likely scenario. Let's leave it out for now but see what happens in research - we have a wide range of users so this may well come up.

Reply:
Hmm. But then… should we be talking about 'backwards compatibility mode' in the kit? 😕

Reply:
Following discussion with @36degrees, I've removed the prototype kit line for now. I've kept it on hand in case we decide to add it back in at any point.